numpy/distutils/system_info.py

```diff
@@ -147,7 +147,7 @@
 
 from numpy.distutils.exec_command import (
     find_executable, filepath_from_subprocess_output,
-    get_pythonexe)
+    )
```

Comment:
Note that changes like this affect our public API - `np.distutils.system_info.get_pythonexe` will fail

Reply:
I thought we were discouraging deep imports, especially for modules with no `__all__`, but I will back this out for now

Reply:
We absolutely should be, but it might be too late